core-libs/setup/ssr/logger/loggers/legacy-server-logger.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { LegacyExpressServerLogger } from './legacy-server-logger';
+
+describe('LegacyServerLogger', () => {
```

Comment:
[MINOR]

Let's adjust the description and the spec file name
```suggestion
describe('LegacyExpressServerLogger', () => {
```